src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const allSpans = [...document.getElementsByClassName('population')];
+let totalValue = 0;
+
+for (let i = 0; i < allSpans.length; i++) {
+  const num = +allSpans[i].textContent.split(',').join('');
```

Comment:
The conversion of text content to a number is done correctly by removing commas. However, consider adding a check to ensure that the text content is a valid number before conversion to avoid NaN values.